libs/components/forms/src/lib/modules/input-box/input-box.component.html

```diff
@@ -170,16 +170,10 @@
 
 <ng-template #errorLabelTemplate>
   <div class="sky-control-errors" role="alert" [id]="errorId">
-    <sky-input-box-errors
-      *ngIf="
-        labelText &&
-        controlDir &&
-        controlDir.invalid &&
-        (controlDir.dirty || controlDir.touched) &&
-        controlDir.errors
-      "
-      [errors]="controlDir.errors"
+    <sky-errors
+      [errors]="controlDir?.errors"
       [labelText]="labelText"
+      [showErrors]="controlDir?.touched || controlDir?.dirty"
```

Comment:
What about `controlDir.invalid`?

Reply:
The previous logic for input box used `touched` or `dirty` to prevent errors on fields before they have been acted on.